### PR TITLE
DevTools Tips 23: Source maps doc refresh

### PR DESCRIPTION
**Pull request by sofiayem**
Source map doc refresh. Relates to #5673 but doesn't fix it yet. Expect follow-ups.

No blockers, merging as soon as reviewed.

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/javascript/source-maps/index.md [hunk 1]
```diff
@@ -1,98 +1,83 @@
...
-You need to use a minifier that's capable of creating source maps.
-For an extended view, see the [Source maps: languages,
-tools and other info][4] wiki page.
-
-The following types of preprocessors are commonly used in combination with Source Maps:
+The following types of preprocessors are commonly used in combination with source maps:
 
 - Transpilers ([Babel][5], [Traceur][6])
 - Compilers ([Closure Compiler][7], [TypeScript][8], [CoffeeScript][9], [Dart][10])
 - Minifiers ([UglifyJS][11])
```
Comment (jecfish) [inline site/en/docs/devtools/javascript/source-maps/index.md:31]:
```suggestion
- Minifiers ([UglifyJS][11])
```

UglifySJ is no longer maintain, use https://github.com/terser/terser

File: site/en/docs/devtools/javascript/source-maps/index.md [hunk 2]
```diff
@@ -1,98 +1,83 @@
...
 
-You need to use a minifier that's capable of creating source maps.
-For an extended view, see the [Source maps: languages,
-tools and other info][4] wiki page.
-
-The following types of preprocessors are commonly used in combination with Source Maps:
+The following types of preprocessors are commonly used in combination with source maps:
 
 - Transpilers ([Babel][5], [Traceur][6])
 - Compilers ([Closure Compiler][7], [TypeScript][8], [CoffeeScript][9], [Dart][10])
```
Comment (jecfish) [inline site/en/docs/devtools/javascript/source-maps/index.md:30]:
```suggestion
- Compilers ([TypeScript][8], [Dart][10])
```
Closure Compiler is not maintain soon as well, remove it, Coffees cript is old so remove as well. Remove the reference URLs, Check if Dart URL updated.

File: site/en/docs/devtools/javascript/source-maps/index.md [hunk 3]
```diff
@@ -1,98 +1,83 @@
...
 - Minifiers ([UglifyJS][11])
```
Comment (jecfish) [inline site/en/docs/devtools/javascript/source-maps/index.md:32]:
```suggestion
```

Add bundlers: Webpack, Vite, esbuild

**Review by jecfish — CHANGES_REQUESTED** [review]

File: site/en/docs/devtools/javascript/source-maps/index.md [hunk 4]
```diff
@@ -24,11 +24,14 @@ To use source maps:
 
 ## Use a supported preprocessor {: #use_a_supported_preprocessor }
 
-The following types of preprocessors are commonly used in combination with source maps:
+The following types of preprocessors are commonly :
 
-- Transpilers ([Babel][5], [Traceur][6])
-- Compilers ([Closure Compiler][7], [TypeScript][8], [CoffeeScript][9], [Dart][10])
-- Minifiers ([UglifyJS][11])
+Common preprocessors used in combination with source maps include but aren't limited to:
```
Comment (jecfish) [inline site/en/docs/devtools/javascript/source-maps/index.md:29]: @jarin, is this list and the categorization correct?

File: site/en/docs/devtools/javascript/source-maps/index.md [hunk 5]
```diff
@@ -1,105 +1,110 @@
...
-2.  When stepping through code (the links in the call stack should open the original source file)
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/Ny275qI4jkgZaZpM1nmU.png", alt="The deployed file with the sourceMappingURL comment.", width="800", height="498" %}
 
-## @sourceURL and displayName {: #sourceurl_and_displayname }
+When you open any deployed file, DevTools notifies you if it found the `//# sourceMappingURL` comment and the associated original file.
 
-While not part of the Source Map spec, the `@sourceURL` allows you to make development much easier
-when working with evals. This helper looks very similar to the `//# sourceMappingURL` property and
-is actually mentioned in the Source Map V3 specifications.
+Notice that the **Editor** automatically pretty-printed the deployed file. In fact, it contains all the code in a single line, except for the `//# sourceMappingURL` comment.
```
Comment (jaro-sevcik) [inline site/en/docs/devtools/javascript/source-maps/index.md:85]: "In fact" -> "In reality" ?

**Review by jaro-sevcik — COMMENTED** [review]
This is looking great. I am wondering if it makes sense to get into more details for some use cases or pitfalls.

For example, it is possible to attach source map manually from the context menu, but that workflow is not documented (as far as I know).

For an example of a recently reported pitfall, a developer correctly served the source map from their server, but did not serve their source code, so they got a [weird error message](https://stackoverflow.com/questions/75816192/cant-set-breakpoint-in-js-file-in-chrome-dev-tools-could-not-load-content-for). (Arguably, we should improve that error message.)

**Pull request merged**